py3status/modules/dropboxd_status.py

```diff
@@ -3,55 +3,87 @@
 Display status of Dropbox daemon.
 
 Configuration parameters:
-    cache_timeout: how often we refresh this module in seconds (default 10)
-    format: prefix text for the dropbox status (default 'Dropbox: {}')
+    cache_timeout: refresh interval for this module (default 10)
+    format: display format for this module (default "Dropbox: {status}")
+    status_busy: show when Dropbox is busy (default None)
+    status_off: show when Dropbox isn't running (default "isn't running")
+    status_on: show when Dropbox is up to date (default "Up to date")
 
-Valid status values include:
+Value for `status_on` if left None:
     - Dropbox isn't running!
+Value for `status_off` if left None:
+    - Up to date
+Values for `status_busy` if left None:
+    - Connecting...
```

Comment:
I don't really like this whole section. `Value for `status_on` if left None:` reads more as code than as documentation.

Not sure how to do this best.